packages/app/.watchmanconfig

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
remove or add conf

Reply:
Having this file is the default setup at rn